spec/lib/http/form_data/multipart_spec.rb

```diff
@@ -95,4 +95,13 @@ def disposition(params)
     subject { form_data.content_length }
     it { is_expected.to eq form_data.to_s.bytesize }
   end
+
+  describe "#boundary" do
+    subject { form_data.boundary }
+    it { is_expected.not_to be_empty }
+
+    it "is included in content type" do
+      expect(form_data.content_type).to end_with(form_data.boundary)
```

Comment:
Sure. I mean `#content_type` already contains this particular test. It's just a bit more verbose there (obviously). And it needs to utilize `boundary `getter (right now it uses `instance_variable_get` as far as I remember)